cnf-certification-test/performance/suite.go

```diff
@@ -0,0 +1,106 @@
+	"github.com/sirupsen/logrus"
+	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/common"
+	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/identifiers"
+	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/results"
+	"github.com/test-network-function/cnf-certification-test/internal/clientsholder"
+	"github.com/test-network-function/cnf-certification-test/internal/crclient"
+	"github.com/test-network-function/cnf-certification-test/pkg/provider"
+	"github.com/test-network-function/cnf-certification-test/pkg/scheduling"
+	"github.com/test-network-function/cnf-certification-test/pkg/testhelper"
+)
+
+// All actual test code belongs below here.  Utilities belong above.
+var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
+	logrus.Debugf("Entering %s suite", common.PerformanceTestKey)
+	var env provider.TestEnvironment
+	ginkgo.BeforeEach(func() {
+		env = provider.GetTestEnvironment()
+	})
+	ginkgo.ReportAfterEach(results.RecordResult)
+
+	testID, tags := identifiers.GetGinkgoTestIDAndLabels(identifiers.TestSchedOtherInSharedCpuPool)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		var nonGuaranteedPodContainers []*provider.Container
+		for _, pod := range env.GetNonGuaranteedPods() {
+			nonGuaranteedPodContainers = append(nonGuaranteedPodContainers, pod.Containers...)
+		}
+		testhelper.SkipIfEmptyAll(ginkgo.Skip, nonGuaranteedPodContainers)
+		testSchedOtherPolicyInSharedCPUPool(&env, nonGuaranteedPodContainers)
+	})
+})
+
+func testSchedOtherPolicyInSharedCPUPool(env *provider.TestEnvironment,
+	nonGuaranteedPodContainers []*provider.Container) {
+	for _, testContainer := range nonGuaranteedPodContainers {
+		logrus.Infof("processing container %v", testContainer)
+		// Get the container pid
+		nodeName := testContainer.NodeName
+		debugPod := env.DebugPods[nodeName]
+		if debugPod == nil {
+			ginkgo.Fail(fmt.Sprintf("Debug pod not found on Node: %s", nodeName))
+		}
+		ocpContext := clientsholder.NewContext(debugPod.Namespace, debugPod.Name, debugPod.Spec.Containers[0].Name)
+		pid, err := crclient.GetPidFromContainer(testContainer, ocpContext)
+		if err != nil {
+			logrus.Errorf("unable to get container process id due to: %v", err)
+			continue
+		}
+		logrus.Infof("obtained process id for %s is %d", testContainer, pid)
+
+		// Get the pid namespace
+		pidNamespace, err := crclient.GetContainerPidNamespace(pid, testContainer)
+		if err != nil {
+			logrus.Errorf("unable to get pid namespace due to: %v", err)
+		}
+		logrus.Infof("obtained pidNamespace for %s is %s", testContainer, pidNamespace)
+
+		// Get the list of process ids running in the pid namespace
+		pids := crclient.GetPidsFromPidNamespace(pidNamespace, testContainer)
+
+		// Check for the specified priority for each processes running in that pid namespace
+		for _, pid := range pids {
+			logrus.Infof("checking the scheduling policy for pid=%d", pid)
```

Comment:
use tnf.ClaimFilePrintf to print "curated" logs both on console and in claim file. Move other logs to debug level. Log here should be including info about namespace/pod/container/pid namespace/pid/policy/priority

Reply:
It's not clear to me what to print with Debug level and what with ClaimFilePrintf. Could we discuss that?